pages/docs/reference/java-to-kotlin-interop.md

```diff
@@ -333,24 +333,28 @@ making its getter and setter methods static members in that object or the class
 > Default methods are available only for targets JVM 1.8 and above.
 {:.note}
 
-> The `@JvmDefault` annotation is experimental in Kotlin 1.3. Its name and behavior may change, leading to future incompatibility.
-{:.note}
-
 Starting from JDK 1.8, interfaces in Java can contain [default methods](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html).
-You can declare a non-abstract member of a Kotlin interface as default for the Java classes implementing it.
-To make a member default, mark it with the [`@JvmDefault`](/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default/index.html) annotation.
+To make all non-abstract members of Kotlin interfaces default for the Java classes implementing them, compile the Kotlin 
+code with the `-Xjvm-default=all` compiler option.
+
 Here is an example of a Kotlin interface with a default method:
 
 <div class="sample" markdown="1" theme="idea" data-highlight-only>
 
 ```kotlin
+// compile with -Xjvm-default=all
+
 interface Robot {
-    @JvmDefault fun move() { println("~walking~") }
+    fun move() { println("~walking~") }  // will be default in the Java interface
     fun speak(): Unit
 }
 ```
 </div>
 
+>**Note**: Compiling with `-Xjvm-default=all` generally works as if you annotated all non-abstract methods of interfaces
```

Comment:
I think we should rather extract this into a separate section named `@JvmDefault` in the end of this chapter, which would clearly say that "previously, to generate default methods you had to use this annotation, but it's not needed anymore, -Xjvm-default=all does it for you; JvmDefault is deprecated now", etc.

Reply:
I don't really like the idea of creating a dedicated section for `@JvmDefault`. People can think that it's an equal alternative, not an experimental mechanism from the past. 
Additionally, we'll have to support its anchor URL after removing it when we deprecate the annotation (in a couple of months, right?).
I'd put the text you suggest in a note (so that readers distinguish it from the common text flow) and provide a link to Sveta's blog post about how this all changes in 1.4.

Reply:
Great, thanks :)

> (in a couple of months, right?)

Yes, https://youtrack.jetbrains.com/issue/KT-40392.